app/helpers/training_routes_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module TrainingRoutesHelper
```

Comment:
Refactor - taking logic out of view now it's getting a bit more complicated re different hints.